### PR TITLE
Add jandi :latest for JANDI.app

### PR DESCRIPTION
**Pull request by weihanglo**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/jandi.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  auto_updates true
+
+  app 'JANDI.app'
+
+  uninstall quit: 'com.jandi.osx.JANDI'
+
+  zap delete: [
+                '~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/com.jandi.osx.jandi.sfl',
+                '~/Library/Application Support/JANDI',
+                '~/Library/Application\ Support/com.jandi.osx.JANDI.ShipIt',
```
Comment (jawshooah) [inline Casks/jandi.rb]: No need for the `\`

File: Casks/jandi.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+cask 'jandi' do
+  version :latest
+  sha256 :no_check
+
+  # cloudfront.net was verified as official when first introduced to the cask
```
Comment (jawshooah) [inline Casks/jandi.rb]: Should include the subdomain: `d1fxzwizdp8gty.cloudfront.net`

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 20c1eff886e0594ddfb702d647f1dae4923efc09. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by weihanglo** [comment]
Thanks for accepting.

**Pull request closed without merging**